x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials.tsx

```diff
@@ -52,74 +49,72 @@ export const Credentials: React.FC = () => {
   }, []);
 
   return (
-    <>
-      <SetPageChrome trail={[CREDENTIALS_TITLE]} />
-      <EuiPageHeader pageTitle={CREDENTIALS_TITLE} />
-      <EuiPageContentBody>
-        {shouldShowCredentialsForm && <CredentialsFlyout />}
-        <EuiPanel hasBorder className="eui-textCenter">
-          <EuiTitle size="s">
+    <AppSearchPageTemplate
+      pageChrome={[CREDENTIALS_TITLE]}
+      pageHeader={{ pageTitle: CREDENTIALS_TITLE }}
+    >
+      {shouldShowCredentialsForm && <CredentialsFlyout />}
+      <EuiPanel color="subdued" className="eui-textCenter">
```

Comment:
2 quick notes,

1. This large diff block is mostly whitespace diff shenanigans, so don't forget to disable those
2. I did a general panel pass with Davey in light of the new page template header having a bottom border, and he approved converting this panel from bordered to subdued